src/server/db/schema.ts

```diff
@@ -114,16 +119,73 @@ export const videoEntries = mysqlTable("videoEntry", {
   uploadId: varchar("uploadId", { length: 256 }).notNull(),
   assetId: varchar("assetId", { length: 255 }),
   downloadUrl: varchar("url", { length: 256 }),
-  playbackId: varchar("playbackId", { length: 256 }),
-  // projectId: bigint("projectId", { mode: "number" }).references(() => project.id),
-  authorId: varchar("authorId", { length: 255 })
+});
+
+export const organizations = mysqlTable("organization", {
+  id: bigint("id", { mode: "number" }).primaryKey().autoincrement(),
+  name: varchar("name", { length: 128 }).notNull(),
+  owner: varchar("ownerId", { length: 255 })
     .notNull()
     .references(() => users.id),
 });
 
-export const videoEntriesRelations = relations(videoEntries, ({ one }) => ({
-  author: one(users, {
-    fields: [videoEntries.authorId],
-    references: [users.id],
+export const organizationsRelations = relations(
+  organizations,
+  ({ one, many }) => ({
+    owner: one(users, {
+      fields: [organizations.owner],
+      references: [users.id],
+    }),
+    usersToOrganizations: many(usersToOrganizations),
   }),
+);
+
+export const usersToOrganizations = mysqlTable(
+  "usersToOrganizations",
+  {
+    memberId: varchar("memberId", { length: 255 })
+      .notNull()
+      .references(() => users.id),
+    organizationId: bigint("organizationId", { mode: "number" })
+      .notNull()
+      .references(() => organizations.id),
+  },
+  (t) => ({ pk: primaryKey(t.memberId, t.organizationId) }),
+);
+
+export const usersToOrganizationsRelations = relations(
+  usersToOrganizations,
+  ({ one }) => ({
+    organization: one(organizations, {
+      fields: [usersToOrganizations.organizationId],
+      references: [organizations.id],
+    }),
+    user: one(users, {
+      fields: [usersToOrganizations.memberId],
+      references: [users.id],
+    }),
+  }),
+);
+
+export const projects = mysqlTable("project", {
+  id: bigint("id", { mode: "number" }).primaryKey().autoincrement(),
+  projectName: varchar("projectName", { length: 256 }).notNull(),
+  projectDescription: varchar("projectDescription", { length: 512 }),
+  title: varchar("title", { length: 256 }),
+  description: varchar("description", { length: 512 }),
```

Comment:
What's the difference between `projectDescription` and `description`?

Reply:
In `projectDescription` users in organization can put whatever they want, `description` is a part of YT metadata. Can be merged into one, idc

Reply:
Leave it